app/assets/javascripts/student_profile/record_service.js

```diff
@@ -197,7 +197,7 @@
           onClick: this.onClickCancel
         }, 'Cancel'),
         (this.props.requestState === 'pending') ? dom.span({}, 'Saving...') : null,
-        (this.props.requestState === 'error') ? dom.span({}, 'Try again!') : null
+        dom.span({}, this.props.requestState)
```

Comment:
What will this do when `this.props.requestState === 'pending'`? Off-the-cuff, looks like it will display 'Saving...' on top and 'pending' on the bottom, which isn't what we want.